hyperspy/misc/eds/utils.py

```diff
@@ -243,6 +247,72 @@ def take_off_angle(tilt_stage,
                                   + math.sin(a) * math.sin(c)))
 
 
+def xray_lines_model(elements=['Al', 'Zn'],
```

Comment:
I have absolutely no idea why these were the defaults -- I'm going to set elements=None and weight_percents=None.

The only more intelligent thing I can think of is to read the elements from the metadata i.e. s.metadata.Sample.elements - but probably many people won't have set that and it will just cause an error -- on which point do you think I should add 

if elements is None:
raise Warning (no elements specified - please specify some)